backend/src/main/java/c/team/session/model/Session.java

```diff
@@ -0,0 +1,24 @@
+package c.team.session.model;
+
+
+import c.team.account.model.UserAccount;
+import c.team.message.model.Message;
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@Builder
+public class Session {
```

Comment:
Myślałem, że trzeba dodać `@Document`, żeby zapisywało się w bazie?

Reply:
Ty kurka faktycznie, nie zauważyłem, że umknęło - ale jak testowałem to dało radę załadować sesję do bazy i wyciągnąć z niej, niemniej jednak lepiej trzymać się jednego mechanizmu działania - dodam to.